ModelComputeCluster.py

```diff
@@ -65,6 +73,17 @@ def __init__(self, csv_parser: CSVLogParser, config_filename: str, output_dir: s
         '''
         super().__init__(csv_parser, config_filename, output_dir, starttime, endtime, queue_filters, project_filters)
 
+        if self._starttime_dt:
```

Comment:
Move to base class to start time and first hour printed together and are set consistently.